scripts/en2ja.json

```diff
@@ -0,0 +1,104 @@
+    },
+    {
+      "from": "rel=\"canonical\" href=\"https://help.okta.com/oag/en-us",
+      "to": "rel=\"canonical\" href=\"https://help.okta.com/oag/ja-jp",
+      "note": "Many (but not all) topics contain a canonical link in the header."
+    },
+    {
+      "from": "rel=\"canonical\" href=\"https://help.okta.com/oie/en-us",
+      "to": "rel=\"canonical\" href=\"https://help.okta.com/oie/ja-jp",
+      "note": "Many (but not all) topics contain a canonical link in the header."
+    },
+    {
+      "from": "rel=\"canonical\" href=\"https://help.okta.com/wf/en-us",
+      "to": "rel=\"canonical\" href=\"https://help.okta.com/wf/ja-jp",
+      "note": "Many (but not all) topics contain a canonical link in the header."
+    },
+    {
+      "from": "\"https://help.okta.com/en-us/Content/index.htm\"",
+      "to": "\"https://help.okta.com/ja-jp/Content/index.htm\"",
+      "note": "This link is in the top-level menu bar in all topics."
+    },
+    {
+      "from": "\"https://help.okta.com/en-us/Content/Topics/ReleaseNotes/okta-relnotes.htm\"",
+      "to": "\"https://help.okta.com/ja-jp/Content/Topics/ReleaseNotes/okta-relnotes.htm\"",
+      "note": "This link is in the top-level menu bar in all topics."
+    },
+    {
+      "from": "\"https://support.okta.com/help/s/\"",
+      "to": "\"https://support.okta.com/help/s/?language=ja\"",
+      "note": "Apply query param to direct users to their language."
+    },
+    {
+      "from": "?language=en_US",
+      "to": "?language=ja",
+      "note": "Query param as used by various Okta Support assets. Appears in multiple links."
+    },
+    {
+      "from": "alt=\"Okta Support\"",
+      "to": "alt=\"Oktaサポート\"",
+      "note": "The 'alt' attr on the top-mast Okta H.O.C. banner."
+    },
+    {
+      "from": "title=\"Okta Support\"",
+      "to": "title=\"Oktaサポート\"",
+      "note": "The 'title' attr on the top-mast Okta H.O.C. banner."
+    },
+    {
+      "from": "aria-label=\"Show Navigation Panel\"",
+      "to": "aria-label=\"ナビゲーションパネルを表示\""
+    },
+    {
+      "from": "aria-label=\"Search Field\"",
+      "to": "aria-label=\"検索フィールド\""
+    },
+    {
+      "from": "aria-label=\"Search Filter\"",
+      "to": "aria-label=\"検索フィルタ\""
+    },
+    {
+      "from": "aria-label=\"Breadcrumbs\"",
+      "to": "aria-label=\"パンくず\""
+    },
+    {
+      "from": "aria-label=\"Change language\"",
+      "to": "aria-label=\"言語を変更\""
+    },
+    {
+      "from": "aria-label=\"言語の変更\"",
+      "to": "aria-label=\"言語を変更\"",
+      "note": "Vendor revised this translation (10/2022). This string is in acceptance tests."
+    },
+    {
+      "from": "aria-label=\"Navigate previous\"",
+      "to": "aria-label=\"前へ移動\""
+    },
+    {
+      "from": "aria-label=\"Navigate next\"",
+      "to": "aria-label=\"次へ移動\""
+    },
+    {
+      "from": "data-mc-back-link=\"Back\"",
+      "to": "data-mc-back-link=\"戻る\"",
```

Comment:
where is this used? So sad that MadCap generates this... Ideally we either ignore it, ask MadCap to not generate it if it is not needed, or we have XTM translate it for us

Reply:
I called MadCap Flare Support. They couldn't tell me what this attribute is used for, in essence they said it's just build cruft, and there's nothing we can do about it :face_palm: 
I'll revert this rule, it might give the wrong impression if we translate it (like there's a "back" link or something)